src/main/java/com/rakovets/course/java/core/practice/array/Task01.java

```diff
@@ -32,9 +35,17 @@ public static void main(String[] args) {
      * @return средняя арифметическая отметка
      */
     static double getAverageMark(int[] marks) {
-        //TODO
-        // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return 0.0;
+        double averageMark = 0.0;
+        double sumMark = 0;
+        int i = 0;
+
+        for ( ; i < marks.length; i++) {
+            sumMark += marks[i];
+        }
+        averageMark = (double) (sumMark / i);
```

Comment:
```suggestion
        averageMark = (double) (sumMark / array.length());
```
Best way: using `array.length()`